app/models/activity.rb

```diff
@@ -346,6 +340,36 @@ def funding_organisation
     service_owner
   end
 
+  def accountable_organisation
+    return service_owner if fund? || programme?
+
+    extending_organisation.is_government? ? service_owner : extending_organisation
+  end
+
+  def accountable_organisation_name
```

Comment:
These getters aren't being tested right now, is that ok?

Reply:
Good point! I've pushed an update which adds a spec for each of the getters